packages/glimmer-demos/lib/ripples.ts

```diff
@@ -76,21 +76,17 @@ export function start() {
   });
 
   document.addEventListener("mousemove", function(e) {
-    data.points.push( new Point(e.timeStamp, e.offsetX, e.offsetY) );
+    data.points.push( new Point(window.performance.now(), e.layerX, e.layerY) );
```

Comment:
This fixes a bug in firefox where the `offsetX` and `offsetY` values were incorrect for many sections of the page. ([layerX](https://developer.mozilla.org/en-US/docs/Web/API/UIEvent/layerX) is deprecated, although it works in all browsers that I tried)